### PR TITLE
D1b

### PR DESCRIPTION
**Pull request by Oddvocado**

### CODE REVIEW

File: src/UDS.Net.API/Entities/D1b.cs [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+        [Comment("7a3c. Macrohemorrhage(s)")]
+        public int? IMAGMACH { get; set; }
+
+        [Comment("7a3d. Microhemorrhage(s)")]
+        public int? IMAGMICH { get; set; }
+
+        [Comment("7a3e. Moderate white-matter hyperintensity (CHS score 5-6)")]
+        public int? IMAGMWMH { get; set; }
+
+        [Comment("7a3f. Extensive white-matter hyperinensity (CHS score 7-8+)")]
```
Comment (mlan225) [inline src/UDS.Net.API/Entities/D1b.cs:133]: Looks like the documentation has a spelling error in the question. Couldn't hurt to fix this up for when we copy the text over to the web project to use as labels

File: src/UDS.Net.API/Entities/D1b.cs [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+        public int? TRACEROTH { get; set; }
+
+        [Comment("6d4a. Other tracer-based imaging - Consistent with other etiology (specify)")]
+        [MaxLength(60)]
+        public string? TRACEROTHX { get; set; }
+
+        [Comment("7a. Structural Imaging (i.e., MRI or CT) - Was structural imaging data or information used to support an etiological diagnosis?")]
+        public int? STRUCTDX { get; set; }
+
+        [Comment("7a1. Atrophy pattern conistent with AD")]
```
Comment (mlan225) [inline src/UDS.Net.API/Entities/D1b.cs:109]: Another spelling error in the documentation, so not your fault at all

conistent -> consistent

**Comment by mlan225** [comment]
Looks pretty good, the only thing i could find were spelling errors in the documentation, so it's definitely not your fault.

But while we are here, we might as well fix em up since we'll most likely use these comments to copy over and use as labels for the web project (at least that's how I do it 👀 )

**Comment by Oddvocado** [comment]
Good catch @mlan225 ! I have gone ahead and corrected the typos.

**Comment by mlan225** [comment]
Sweet @Oddvocado looks good to me 👍

**Pull request merged**